### DISCUSSION TITLE
Putting an image to a tablet

### DISCUSSION THREAD

**Discussion by glic4** [lllyasviel/Fooocus#1492] [Q&A]
Hi, Is there any way to put an image (for example taken from the screenshot of the real application) to another image, a person who holds a tablet? The output i am expecting is a person who is looking at the tablet with a screen of my application. Do You have any ideas how can i get this?

**Comment by mkfoto** [comment]
Generate the image you want in fooocus then swap the screen image in Photoshop. Not sure you can do it in fooocus alone.
I tried "woman standing on street, view from behind, holding and looking at an ipad. Weather report on screen". This generated a few usable images.
